packages/components/src/tag/tests/chromatic/TagList.chroma.jsx

```diff
@@ -0,0 +1,189 @@
+import { Counter } from "@components/counter";
```

Comment:
This file should not be added. Its been renamed to .stories.tsx and this file added is the result of a merge conflict gone wrong